pkgs/tools/filesystems/android-file-transfer/default.nix

```diff
@@ -1,22 +1,26 @@
 { lib, stdenv, fetchFromGitHub, cmake, fuse, readline, pkgconfig, qtbase }:
 
 stdenv.mkDerivation rec {
-  name = "android-file-transfer-${version}";
-  version = "3.7";
+  pname = "android-file-transfer";
+  version = "3.8";
+
   src = fetchFromGitHub {
     owner = "whoozle";
     repo = "android-file-transfer-linux";
     rev = "v${version}";
     sha256 = "0a388pqc0azgn0wy85wb1mjk3b5zb6vcr58l4warwfzhca400zn0";
   };
+
   buildInputs = [ cmake fuse readline pkgconfig qtbase ];
+
   buildPhase = ''
     cmake .
     make
   '';
   installPhase = ''
     make install
```

Comment:
That's the default install target so the phase overriding is probably unnecessary here